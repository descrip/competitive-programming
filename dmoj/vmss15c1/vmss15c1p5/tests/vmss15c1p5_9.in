10 718 953
candy 531 3 7
cigarette 936 1 4
dynamicprogramming 483 20 29
goldendelicious 449 25 13
jemand 558 22 2
laptop 425 25 3
leo 836 16 5
mountain 974 25 31
partimejob 908 18 29
rhode 760 11 23
