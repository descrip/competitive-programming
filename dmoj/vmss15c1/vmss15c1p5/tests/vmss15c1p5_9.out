224748
candy 0
cigarette 227
dynamicprogramming 0
goldendelicious 0
jemand 22
laptop 0
leo 0
mountain 0
partimejob 0
rhode 0
